english/marshmallow.md

```diff
@@ -694,3 +694,8 @@ I consider publishing a spoiler to the new [Star Wars movie](star-wars-spoiler/s
 "What's a Heffelump?" asked Sheffield inquisitively.
 
 "I dont't know! Why should I care?" replied Donald Trump.
+
+When faced with a lot of work and impending deadlines in the (not so near) future,
+there are two options you must choose from. You may either do [this](gtd/gtd.md), or [that](procrastinate/procrastinate.md).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.